### DISCUSSION TITLE
Fail to enable preview-tui in wsl2 Ubuntu

### DISCUSSION THREAD

**Discussion by GKarbon** [jarun/nnn#1581] [Q&A] [answered]
Hi, I installed nnn on wsl2 Ubuntu and running in Windows Terminal.
I want to active preview-tui plugin, but it just show an error.

![Error](https://user-images.githubusercontent.com/118652369/218265535-69f74711-11ca-477c-b76a-abd44810fcbf.png)

Do I need to change something in the script?

**Comment by luukvbaal** [comment, marked as answer]
I encountered this as well while testing WSL again. It's fixed in https://github.com/jarun/nnn/pull/1580.

**Reply by GKarbon** [reply]
Yes, that solved my problem, thanks.
